bentoml/_internal/bento/bento.py

```diff
@@ -144,13 +143,14 @@ def create(
 
         # Create bento.yaml
         with bento_fs.open(BENTO_YAML_FILENAME, "w") as bento_yaml:
-            BentoInfo.from_build_args(tag, svc, labels, models).dump(bento_yaml)
+            # pyright doesn't know about attrs converters
+            BentoInfo(tag, svc, labels, models).dump(bento_yaml)  # type: ignore
 
-        return OSBento(tag, bento_fs)
+        return SysPathBento(tag, bento_fs)
 
     @classmethod
-    def from_fs(cls, tag: Tag, fs: FS) -> "Bento":
-        res = cls(Tag("", ""), fs)
+    def from_fs(cls, item_fs: FS) -> "Bento":
+        res = cls(None, item_fs)  # type: ignore
```

Comment:
```suggestion
        res = cls(None, item_fs)  # type: Bento
```

Reply:
The issue is the `None` tag argument, not the result type of the expression.

Reply:
why we cannot make this class `tag: t.Optiional[Tag]` here as usual?